src/test/java/com/jcabi/aether/MavenClasspathTest.java

```diff
@@ -55,35 +56,76 @@
 public final class MavenClasspathTest {
 
     /**
+     * Group of test artifact.
+     */
+    private static final String GROUP = "junit";
```

Comment:
@mkordas I need that constat either way, because its used twice for 1 test anyway. As for the second test it can be any dependency, the point is the variable parameter should work and not crash somehow the logic.